### ISSUE TITLE
"500" Internal Server Error when trying to access config pannel from webadmin

### ISSUE THREAD

**Issue by obrenckle** [YunoHost-Apps/my_webapp_ynh#97]
### Describe the bug

After installing the app (successfully, I think), when I try to access the app config pannel from yunohost webadmin, I get an error message :
Erreur: "500" Internal Server Error
Action: "GET" /yunohost/api/apps/my_webapp/config-panel?full&locale=fr
Message d'erreur :
Config panel question 'fpm_free_footprint' should be initialized with a value during install or upgrade.

### Context

- Hardware: *Other ARM board (Olimex lime2)*
- YunoHost version: 11.0.10.2 (stable)
- I have access to my server: Through SSH, through the webadmin and direct access via keyboard / screen
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: yunohost installed on top of armbian, because there was no armbian 11 image at the time
- Using, or trying to install package version/branch: 1.0~ynh11

### Steps to reproduce

   1. Go to webadmin
   2. Go to Application / My_Webapp_label
   3. Click on Config pannel
   4. Get error message

### Expected behavior

I expect access to a config pannel (to modify php / sql settings maybe ?)

### Logs

Erreur: "500" Internal Server Error
Action: "GET" /yunohost/api/apps/my_webapp/config-panel?full&locale=fr
Message d'erreur :
Config panel question 'fpm_free_footprint' should be initialized with a value during install or upgrade.

**Comment by tituspijean** [comment]
Thank you for your report! I think this is an oversight in the install script. To confirm this, can you run the following command and try the config panel again? `sudo yunohost app upgrade my_webapp -F`

**Comment by obrenckle** [comment]
Yes, that fixed it, I have access to the config panel, thank you !

**Comment by blindi0815** [comment]
So I know this is closed but I just wanted to add that you have to enter the command 

`sudo yunohost app upgrade my_webapp -F`

for every instance of the webapp you are using. So you have to do:

`sudo yunohost app upgrade my_webapp__2 -F`, `sudo yunohost app upgrade my_webapp__3 -F` and so on.

**Comment by tituspijean** [comment]
(It was kind of assumed that the app ID needs to be adjusted for each app, but you note is welcome. 😀)

I'm keeping this issue open while we fix the install script.

**Comment by biphasic** [comment]
ran into the same issue, thanks for the note!

**Comment by selfhoster1312** [comment]
`sudo yunohost app upgrade my_webapp -F` fixed it for me too, thanks. Probably useless to mention but just in case, problem also happens with PHP disabled on the my_webapp settings.
